### PR TITLE
Add template for reserved ip use cases

### PR DESCRIPTION
**Pull request by sebastus**
demonstrates creating a new reserved ip and using an existing reserved ip, each connecting to a load balancer.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@sebastus__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: reservedip/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,151 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {        
+        "Existing_RIP_SubId": {
```
Comment (singhkay) [inline reservedip/azuredeploy.json]: Please camelCase all parameters

File: reservedip/Readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Snippet - Supported Reserved IP Use Cases
+
+This template demonstrates the currently supported use case for Reserved IP.  A Reserved IP is simply a statically allocated Public IP.  
+
+There is currently only one supported use case: assign a Reserved IP to the front end of the Azure Load Balancer.  
+
+This template goes one step further: it both creates a new Reserved IP and assigns it to a load balancer and it uses a previously existing Reserved IP and assigns it to a separate load balancer.
```
Comment (singhkay) [inline reservedip/Readme.md]: After the execution, will the user have 1 new LB and 1 new PIP and the LB will have two PIPs?

Comment (sebastus) [inline reservedip/Readme.md]: there will be 2 LBs, one with newly created PIP and one with previously existing.

Comment (singhkay) [inline reservedip/Readme.md]: I see, can you put this in the readme? atm it could be confusing.

**Comment by singhkay** [comment]
@sebastus  Can you rename this template to 201-reserved-ip? since it showcases a platform capability.

**Comment by singhkay** [comment]
Looks good, merged!

**Comment by sebastus** [comment]
Thanks, Kay.  My plan is to add to the template as new reserved IP features are released, so keep me in the loop!

**Pull request merged**